workers/unity/Packages/io.improbable.gdk.testutils/Editor/Improbable.Gdk.TestUtils.Editor.asmdef

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Improbable.Gdk.TestUtils.Editor",
+    "references": [
+        "Improbable.Gdk.Tools"
+    ],
+    "optionalUnityReferences": [],
```

Comment:
Should this be test assemblies only?

Reply:
Discussed offline - no since Unity treats test assemblies _oddly_ sometimes. Revisit this when 2019.2 adds a bit more granularity to the asmdefs